New endpoint  to show/hide Prioritize button in Front End Side

dmaity-celestialsys opened the pull request:
Added this end point to check the user organization has access to build Prioritize feature or not from Front End side. And also checked the build is already set as a high priority build or not.

pavel-d commented on lib/travis/api/v3/services/build/build_prioritize.rb:
```diff
@@ -0,0 +1,12 @@
+
+    def run
+      build = check_login_and_find(:build)
+      build_priority = build.owner.build_priority?
```
@dmaity-celestialsys, owner may be a user as well, in this case we'll get undefined method error.

dmaity-celestialsys commented:
Yes @pavel-d, we have added additional field inside existing renders. Added `allow_build_priorities` this method to Owner presenter and `priority` method to build.
